tests/wait_until_previous_builds_complete.sh

```diff
@@ -20,7 +20,7 @@ prevBuildStatus=`drone build info --format='{{.Status}}' vmware/vic $(( $DRONE_B
 echo prevBuildStatus $prevBuildStatus
 
 while [[ $prevBuildStatus == *"running"* ]]; do
-    echo "Waiting 5 minutes for previous build to complete";
+    echo "Waiting 5 minutes for build $(( $DRONE_BUILD_NUMBER-$numServers )) to complete";
```

Comment:
I think it'd be good to extract `$(( $DRONE_BUILD_NUMBER-$numServers ))` into a variable and use that variable both here and on line 19 so that the status check and echo'd message are more likely to stay in sync if our server selection logic changes at some point in the future.